pkg/dbs/userdb/db_methods.go

```diff
@@ -385,3 +505,27 @@ func (dbService *UserDBService) SendReminderToConfirmAccountLoop(
 	}
 	return nil
 }
+
+func (dbService *UserDBService) CreateIndexForUser(instanceID string) error {
+	ctx, cancel := dbService.getContext()
+	defer cancel()
+
+	_, err := dbService.collectionRefUsers(instanceID).Indexes().CreateMany(
+		ctx, []mongo.IndexModel{
+			{
```

Comment:
Could you please check here, what other user indexes could make sense. I think it would be good to have one for the accountID, which is used for the login to look up the user.

Reply:
Added index models for all relevant attributes.